### PR TITLE
BT011 - feat: get events - user, past, accepted

### PR DESCRIPTION
**Pull request by mnk-7**
Adding findAllPastAcceptedByUser() method in SingleEventRepo and getAllPastAcceptedEvents() method in SingleEventService and SingleEventController.

### CODE REVIEW

File: src/main/java/pl/envelo/meetek/controller/event/SingleEventController.java [hunk 1]
```diff
@@ -143,4 +143,22 @@ public ResponseEntity<List<SingleEventShortDto>> getAllPublicFutureNotAcceptedEv
...
+    @GetMapping("/past/accepted")
+    @Operation(summary = "Get all past events where the user with given ID confirmed his participation")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Results returned",
+                    content = {@Content(mediaType = "application/json", schema = @Schema(implementation = SingleEventShortDto.class))}),
+            @ApiResponse(responseCode = "404", description = "No event found", content = @Content)})
+    public ResponseEntity<List<SingleEventShortDto>> getAllPastAcceptedEvents(@RequestParam long userId) {
+        List<SingleEvent> events = singleEventService.getAllPastAcceptedEvents(userId);
+        List<SingleEventShortDto> dtoEvents = events.stream()
+                .map(e -> dtoMapperService.mapToSingleEventShortDto(e))
```
Comment (zaborowskim) [inline src/main/java/pl/envelo/meetek/controller/event/SingleEventController.java:156]: zmień map(mapToSingleEventShortDto::dtoMapperService

Comment (mnk-7) [reply src/main/java/pl/envelo/meetek/controller/event/SingleEventController.java]: done

**Review by MaciejSzczepanczyk — APPROVED** [review]

**Review by zaborowskim — APPROVED** [review]

**Review by kniedzwiedzki — APPROVED** [review]
siema

File: src/main/java/pl/envelo/meetek/controller/event/SingleEventController.java (hunk 1 above)
Comment (kniedzwiedzki) [reply src/main/java/pl/envelo/meetek/controller/event/SingleEventController.java]: (y)

**Pull request merged**